packages/react-devtools-shared/src/backend/DevToolsFiberComponentStack.js

```diff
@@ -40,6 +42,8 @@ export function describeFiber(
   } = workTagMap;
 
   switch (workInProgress.tag) {
+    case HostHoistable:
+    case HostSingleton:
```

Comment:
This was missing previously leading to missing frames for html/body which lead to mismatches.